### DISCUSSION TITLE
Code

### DISCUSSION THREAD

**Discussion by s1pl** [discordjs/discord.js#8169] [Q&A] [answered]
Who can give me code for me bot ( I save the bot in Lua)
Please code for avatar

**Comment by kyranet** [comment, marked as answer]
This is a library for JavaScript and TypeScript bots, we do not provide support for Lua.

For other languages, please refer to [Discord's library list](https://discord.com/developers/docs/topics/community-resources#libraries), which includes [Discordia](https://github.com/SinisterRectus/Discordia) for Lua.
